scripts/action/update.sh

```diff
@@ -3,12 +3,16 @@
 version=""
 
 while IFS= read -r -d '' f; do
-  v="$(yq '.jobs.*.steps[].uses | select(. == "super-linter/super-linter*")' "${f}" | sed -e 's/.*@//g')"
+  v="$(yq '.jobs.*.steps[].uses | select(. == "super-linter/super-linter*") | . + " # " + line_comment' "${f}" | sed -e 's/.*@//g')"
   if [ -n "${v}" ]; then
+    if [[ "${v}" == *" # v"* ]]; then
+        v="${v#* # }"
+    fi
```

Comment:
```suggestion
  vs=($(yq '.jobs.*.steps[].uses | select(. == "super-linter/super-linter*") | . + " # " + line_comment' "${f}" | sed -e 's/.*@//g'))
  v="${vs[-1]}"
  if [ -n "${v}" ]; then
```

こうとかどうだろうか。
たしかyqの行でshellcheckでなんか弾かれるとは思うがこの方がわかりやすそう？

Reply:
```suggestion
  mapfile -t vs < <(yq '.jobs.*.steps[].uses | select(. == "super-linter/super-linter*") | . + " # " + line_comment' "${f}" | sed -e 's/.*@//g')
  v="${vs[-1]}"
  if [ -n "${v}" ]; then
```
こうすればshellcheckに怒られないか。

Reply:
別に構わんけど、やる理由って、複数個見つかるかもしれないから、であってる？

Reply:
単純に `#` 含む変数内置換がどうにも分かりづらいというか。

Reply:
yqでコメント付きで1行抜き出せたらいいんだけどねぇ

Reply:
で、yamlっぽく、コメントは#で区切ってるだけなんだけど、別にこだわる必要もないか・・・？

Reply:
```bash
  v="$(yq '.jobs.*.steps[].uses | select(. == "super-linter/super-linter*") | line_comment' "${f}")"
  if  [ -n "${v}" ]; then
    v="$(yq '.jobs.*.steps[].uses | select(. == "super-linter/super-linter*")' "${f}" | sed -e 's/.*@//g')"
  fi
```

かなり愚直な方法だけど、こういう書き方もあるか、とか思った (コメントを抜き出してみて、だめならバージョン指定を抜き出す)。

Reply:
あーこの方がいいか、確かに。

Reply:
コメントに関係ない文字列が入ってたケースがややこしいかも？

Reply:
`if  [[ "${v}" !~ v* ]]; then` 的な感じにすれば問題なさそう？

Reply:
まあvから始まるコメントをたまたまするケースは無視していいか・・・

Reply:
というかバージョンっぽい文字列の全然関係ないコメントのこと考えたら、まあいいかという気分になった。

Reply:
個人的にはこの辺はそんな厳密にやらなくてもいい気はしてる (多分バージョン指定じゃないコメントつけんやろ的な)

Reply:
[`07cb85c` (#1325)](https://github.com/dev-hato/actions-update-gitleaks/pull/1325/commits/07cb85c51f59c20e67a975dd1c3d4235a46c8bc3)

それっぽくした